### PR TITLE
[61633] Add new Room Resource fields

### PR DESCRIPTION
**Pull request by mrashed-dev**
# Description
Adds in the new Room Resource fields brought in with the v2.2 API update. 

# License
I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by yusra-ahmed** [comment]
Do you need to add any tests for these?

**Comment by mrashed-dev** [comment]
> Do you need to add any tests for these?

There weren't any changes needed to these tests but it's a good idea to add some. Will add and request review when it's done, thank you!

File: tests/test_resources.py [hunk 1]
```diff
@@ -14,3 +14,15 @@ def test_all_resources(api_client):
     assert len(resources) == 2
     for resource in resources:
         assert isinstance(resource, RoomResource)
+
+
+@pytest.mark.usefixtures("mock_resources")
+def test_resource(api_client):
```
Comment (mrashed-dev) [inline tests/test_resources.py:20]: @yusra-ahmed It's not much but the endpoint is a very basic `GET` endpoint without an query support!

**Review by yusra-ahmed — APPROVED** [review]
lgtm

**Pull request merged**